app/javascript/packs/articlePage.jsx

```diff
@@ -71,6 +72,31 @@ function showAnnouncer() {
   document.getElementById('article-copy-link-announcer').hidden = false;
 }
 
+// Temporary Ahoy Stats for comment section clicks
+function trackCommentSectionClicks() {
+  document.getElementById('comments')?.addEventListener('click', (event) => {
```

Comment:
This is my naïveté, but does tabbing into the comments section require any consideration?

Reply:
Oh WOW, I hadn't even thought of that! Will definitely add!

Reply:
That's a great point: I'm not sure. @amywtlin: are you concerned with keyboard interactions?

Reply:
If we can include keyboard interactions that'd be great. 

One bonus point is if we can differentiate actual clicks from keyboard interactions, that'd be also nice. To be honest I can't say for sure right now how I would use the data yet, but if it's easy to differentiate, I'd prefer to log them separately. If it's going to be a trouble, never mind!

Thanks for the thoughtful consideration here.

Reply:
I played around with hooking up keyboard interactions in the comments section, but I don't think the data is going to be worthwhile for a couple of reasons:

- the code to hook up event listeners is kind of complex and hacky: I'm guessing which controls will be present in the `comments` section that can have keyboard interactions (mostly links, buttons, and text fields)
- I'm only looking for space or tab interactions, which means filtering out a lot keypresses
- A lot of times the space bar interactions with controls are reported as clicks as well, leading to a double count.

Reply:
Sounds good to me. Thank you!

Reply:
Thanks for the suggestion @aitchiss: I'm going to go the data attribute tracking route!